src/applications/check-in/components/layout/tests/Footer.unit.spec.jsx

```diff
@@ -98,7 +98,19 @@ describe('check-in', () => {
           <Footer isPreCheckIn={false} />
         </CheckInProvider>,
       );
-      expect(component.getByTestId('travel-help-block')).to.exist;
+      expect(component.getByTestId('help-block')).to.exist;
+    });
+    it('Renders HelpBlock for contact-information page', () => {
```

Comment:
another suggestion for describing this test could be "Renders HelpBlock without travel link on non-travel pages"

Reply:
True, since that's what I'm really testing for here